api/envoy/api/v2/#cds.proto#

```diff
@@ -0,0 +1,852 @@
+syntax = "proto3";
```

Comment:
?

Reply:
Yeah, I'm guessing this is an accidental include of an editor temp file.